docs/setup/connect-to-elasticsearch.asciidoc

```diff
@@ -1,108 +1,75 @@
 [[connect-to-elasticsearch]]
 == Add data to {kib}
+++++
+<titleabbrev>Add data</titleabbrev>
+++++
 
-To start working with your data in {kib}, you can:
+The fastest way to add data to {kib} is to use one of the built-in options.
+You can collect data from an app or service
+or upload a file that contains your data.  If you're not ready to use your own data,
+add a sample data set and give {kib} a test drive.
+The built-in options are available on the home page.
 
-* Upload a CSV, JSON, or log file with the File Data Visualizer.
+[role="screenshot"]
+image::images/add-data-home.png[Built-in options for adding data to Kibana:  Add data, Add Elastic Agent, Upload a file]
 
-* Upload geospatial data with the GeoJSON Upload feature.
+[float]
+[[add-data-tutorial-kibana]]
+===  Add data
 
-* Index logs, metrics, events, or application data by setting up a Beats module.
+Want to ingest logs, metrics, security, or application data?
+Install and configure a Beats data shipper or other module to periodically collect the data
+and send it to {es}. You can then use the pre-built dashboards to explore and analyze the data.
 
-* Connect {kib} with existing {es} indices.
+[role="screenshot"]
+image::images/add-data-tutorials.png[Add Data tutorials]
 
-If you're not ready to use your own data, you can add a <<get-started, sample data set>>
-to see all that you can do in {kib}.
+[discrete]
+=== Add Elastic Agent
 
-[float]
-[[upload-data-kibana]]
-=== Upload a CSV, JSON, or log file
+beta[] *Elastic Agent* is a centralized way to set up your data integrations.
+With the *Fleet* UI, you can add
+and manage integrations for popular services and platforms, providing
+an easy way to collect your data. The integrations
+ship with dashboards and visualizations,
+so you quickly get insights on your data.
 
-experimental[]
+To get started, refer to
+{ingest-guide}/ingest-management-getting-started.html[Quick start: Get logs and metrics into the Elastic Stack].
 
-To visualize data in a CSV, JSON, or log file, you can upload it using the File 
-Data Visualizer. On the home page, click *Upload a file*, and 
-then drag your file onto the *File Data Visualizer*. Alternatively, you can open 
-it by navigating to *Machine Learning* from the side navigation and selecting 
+[role="screenshot"]
+image::images/add-data-fleet.png[Add data using Fleet]
 
-*Data Visualizer*.
+NOTE: This feature is not intended for use as part of a
```

Comment:
Not sure I like this note because it's a little misleading. Eventually it *will* be the main way to ingest data, but right now, it's not recommended in a production environment. Maybe qualify it by saying, "The beta release of this feature...."